egs/wsj/s5/utils/mkgraph.sh

```diff
@@ -13,15 +13,17 @@
 # (this is compiled from this repository using Doxygen,
 # the source for this part is in src/doc/graph_recipe_test.dox)
 
+# shellcheck disable=2064
+
 set -o pipefail
 
 tscale=1.0
 loopscale=0.1
 
 remove_oov=false
 
-for x in `seq 4`; do
-  [ "$1" == "--mono" -o "$1" == "--left-biphone" -o "$1" == "--quinphone" ] && shift && \
+for _ in `seq 4`; do
+  [ "$1" == "--mono" ] || [ "$1" == "--left-biphone" ] || [ "$1" == "--quinphone" ] && shift && \
```

Comment:
I'd prefer you replace this with an if-statement- I don't want the reader to have to reason about the precedence and associativity of the || and && opeators.